### DISCUSSION TITLE
Ability to search/filter within the saved sessions list

### DISCUSSION THREAD

**Discussion by hschlossberg** [sienori/Tab-Session-Manager#1053] [Q&A] [answered]
I wish there were an ability to search/filter session names within the saved sessions list.  It can be a long list to look through!

**Comment by sienori** [comment, marked as answer]
You can search for sessions!
![image](https://user-images.githubusercontent.com/29514424/189958453-e1176b37-917f-4cb9-8201-de7ee76a0871.png)

**Reply by prishs** [reply]
is it possible that search looks for url content also.. 
it seems it only searches in tab name & not in url text.. correct me if wrong..

**Comment by hschlossberg** [comment]
Oops -- thanks! I had been looking for a search field but the icon was 
there the whole time!
